docs/decisions/0002-combined-cookiecutter-repo.rst

```diff
@@ -0,0 +1,37 @@
+2. Combined Cookiecutter Repo
+=============================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+There were multiple Python cookiecutters with lots of overlap, and it was difficult to keep even one always up to date.
+
+* cookiecutter-django-app
+* cookiecutter-django-ida
+
+Decision
+--------
+
+TODO
+
+Note: The ``cookiecutter-xblock `` will continue to be its own independent cookiecutter repo.
+
+^ Is this true???
```

Comment:
Yes, xblock was way too different to be included in the layering.

Reply:
👍